### ISSUE TITLE
Black background color appears while slide left transition. 

### ISSUE THREAD

**Issue by follow2vivek** [kalismeras61/flutter_page_transition#5]

**Comment by follow2vivek** [comment]
Black background color appears while slide left transition.

**Comment by kalismeras61** [comment]
I didn't see any issue, Could give us more information? Which type transition you are using?

**Comment by follow2vivek** [comment]
`Navigator.push(context, PageTransition(type: PageTransitionType.rightToLeft, child: TopicScreen()));`

**This is my code.**

`FolderWidget(
              title: '1${index + 1}th Standard',
              onTab: () => Navigator.of(context).push(
                    PageTransition(
                        child: TopicScreen(),
                        type: PageTransitionType.rightToLeft),
                  ),
            )`

**If i go from first screen to second screen it works fine, but if I go second to third screen, black background appears.**

![git](https://user-images.githubusercontent.com/24294081/53542208-5798f380-3b43-11e9-82bc-cc7a4b18ecdd.gif)

**Comment by kalismeras61** [comment]
As i understand when you Topic screen it does. What did you return build method in topic screen?

**Comment by follow2vivek** [comment]
***This is my TopicScreen code.***
```
class TopicScreen extends StatefulWidget {
  @override
  _TopicScreenState createState() => _TopicScreenState();
}

class _TopicScreenState extends State<TopicScreen> {
  var topics = ['Alkaline', 'Organic Chemistry', 'Chemicals', 'Formula'];

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      backgroundColor: Color.colorWhite,
      appBar: AppBar(
        iconTheme: IconThemeData(
          color: Color.colorMatteBlack,
        ),
        backgroundColor: Color.colorWhite,
        title: Text(
          'Topics',
          style: Style.toolbar,
        ),
      ),
      body: Container(
        child: ListView.builder(
          itemCount: topics.length,
          itemBuilder: (BuildContext context, int index) {
            return FolderWidget(
              title: topics[index],
              onTab: () => Navigator.of(context).push(
                    PageTransition(
                      child: ImageScreen(
                        title: topics[index],
                      ),
                      type: PageTransitionType.rightToLeft,
                    ),
                  ),
            );
          },
        ),
      ),
    );
  }

}
```

**Comment by kalismeras61** [comment]
Can you share also Image screen page codes?

**Comment by follow2vivek** [comment]
```
class ImageScreen extends StatefulWidget {
  final String title;

  ImageScreen({Key key, this.title}) : super(key: key);

  @override
  _ImageScreenState createState() => _ImageScreenState();
}

class _ImageScreenState extends State<ImageScreen> {
  var imageUrl = [
    'https://upload.wikimedia.org/wikipedia/commons/f/f8/Diazolidinyl_urea_correct_formula.png',
    'https://i.stack.imgur.com/wu8Z9.png',
    'https://66.media.tumblr.com/6fb4eff59c581a7a2fb17a7c7c340e4e/tumblr_nllcy1LIr01qjn8wzo1_400.png',
    'https://upload.wikimedia.org/wikipedia/commons/f/f8/Diazolidinyl_urea_correct_formula.png',
    'https://i.stack.imgur.com/wu8Z9.png',
    'https://66.media.tumblr.com/6fb4eff59c581a7a2fb17a7c7c340e4e/tumblr_nllcy1LIr01qjn8wzo1_400.png',
    'https://i.stack.imgur.com/wu8Z9.png',
  ];

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      backgroundColor: Color.colorWhite,
      appBar: AppBar(
        iconTheme: IconThemeData(
          color: Color.colorMatteBlack,
        ),
        backgroundColor: Color.colorWhite,
        title: Text(
          widget.title,
          style: Style.toolbar,
        ),
      ),
      body: Container(
        color: Color.colorWhite,
        padding: EdgeInsets.all(10),
        child: StaggeredGridView.countBuilder(
          itemCount: imageUrl.length,
          crossAxisCount: 4,
          crossAxisSpacing: 10,
          mainAxisSpacing: 10,
          itemBuilder: (context, index) {
            return InkWell(
              child: Container(
                decoration: BoxDecoration(
                  color: Color.colorWhite,
                  borderRadius: BorderRadius.circular(10),
                  boxShadow: [
                    BoxShadow(
                        offset: Offset(0.1, 0.1),
                        blurRadius: 2,
                        spreadRadius: 0.5,
                        color: Colors.grey[400])
                  ],
                ),
                child: Hero(
                  tag: "${index}_image",
                  child: Image(
                    image: NetworkImage(imageUrl[index]),
                    fit: BoxFit.contain,
                  ),
                ),
              ),
              onTap: () => Navigator.push(
                    context,
                    MaterialPageRoute(
                      fullscreenDialog: true,
                      builder: (BuildContext context) => ZoomScreen(
                            imageUrl: imageUrl[index],
                            heroTag: "${index}_image",
                          ),
                    ),
                  ),
            );
          },
          staggeredTileBuilder: (int index) =>
              StaggeredTile.count(2, index.isEven ? 2 : 1),
        ),
      ),
    );
  }
}

```

**Comment by kalismeras61** [comment]
Problem fixed! You need to upgrade your version to 1.0.9! Thanks for reporting the problem.

**Comment by follow2vivek** [comment]
### Thank you for updating, now it working fine.
